src/Analyzer/Passes/tests/gtest_analyzer_utils.cpp

```diff
@@ -0,0 +1,58 @@
+#include <Analyzer/Passes/tests/gtest_analyzer_utils.h>
+#include <gtest/gtest.h>
+
+#include <Analyzer/ColumnNode.h>
+#include <Analyzer/FunctionNode.h>
+#include <Analyzer/IdentifierNode.h>
+#include <Analyzer/IQueryTreePass.h>
+#include <Analyzer/QueryTreeBuilder.h>
+#include <Analyzer/Utils.h>
+#include <Common/tests/gtest_global_context.h>
+#include <Common/tests/gtest_global_register.h>
+#include <Parsers/parseQuery.h>
+#include <Parsers/ExpressionListParsers.h>
+
+using namespace DB;
+
+static QueryTreeNodePtr resolveColumn(DataTypePtr type, QueryTreeNodePtr node, std::map<String, QueryTreeNodePtr> & resolved_map, ContextPtr context)
```

Comment:
If you think that these functions will be reused (utils), please add a couple of lines docstring